packages/inference/vitest.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from "vitest/config";
+import { readFileSync } from "node:fs";
+import { join } from "node:path";
+
+export default defineConfig({
+	define: {
+		__TEST_FILES__: JSON.stringify({
+			"cheetah.png": readFileSync(join(__dirname, "test", "cheetah.png")).toString("base64"),
+		}),
+	},
+});
```

Comment:
This is a neat way to test on both browser & front-end. 

Do you think you can change the other HFInference tests to work on the frontend too? :) (audio-classification, ...)